test/unit/app/helpers/world_location_helper_test.rb

```diff
@@ -0,0 +1,102 @@
+require "test_helper"
+
+class WorldLocationHelperTest < ActionView::TestCase
+  extend Minitest::Spec::DSL
+  include WorldLocationHelper
+
+  context "#group_and_sort" do
+    test "groups and sorts correctly where special characters are used anywhere other than the first character" do
+      world_location_1 = create(:world_location, name: "Cow")
+      world_location_2 = create(:world_location, name: "Côt")
+      unsorted = [world_location_1, world_location_2]
+
+      expected_order = [[
+        "C",
+        [
+          world_location_2,
+          world_location_1,
+        ],
+      ]]
+
+      assert_equal expected_order, group_and_sort(unsorted)
+    end
+
+    test "groups and sorts correctly where special characters are used as the first character" do
+      world_location_1 = create(:world_location, name: "England")
+      world_location_2 = create(:world_location, name: "Éire")
+      unsorted = [world_location_1, world_location_2]
+
+      expected_order = [[
+        "E",
+        [
+          world_location_2,
+          world_location_1,
+        ],
+      ]]
+
+      assert_equal expected_order, group_and_sort(unsorted)
+    end
+
+    test "ignores The prefix when sorting" do
```

Comment:
Can we make this name a bit clearer, I only understand it because I know what it means. Maybe just some backticks: "ignores `The` prefix when sorting"

Reply:
Good point.  I've updated this.